SDL2-CS.csproj.nuspec

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>SDL2-CS-Rolling</id>
+    <version>1.0.0-alpha%timestamp%</version>
```

Comment:
Is there a way to change this to just a ‘yymmdd’ number, or a Git hash?

Reply:
I wanted to have a time-based version that was able to handle multiple pushes a day.. commit hashes aren't sortable so it would be harder to know which is the latest version just by looking.

Reply:
Also, without the `-alpha` I wasn't able to add the package to nuget.org.

Reply:
Man, NuGet is really bad, innit.

Basically I’m just trying to get dumb people to not file reports with stuff like “I’m on 1.0” because that’s totally meaningless for SDL2#, so I want to acoid giving them an inch and having them take a yard...

Does yymmdd-rev make more sense? That’s closer to what Linux packages do where the rev is usually just 0 unless there’s more than one patch added to the stock number.

Reply:
I'm not sure if I have a [variable](https://docs.gitlab.com/ee/ci/variables/#predefined-variables-environment-variables) to use that functions that way. I'm almost positive not using some form of `major.minor.patch-tag` will cause nuget to reject the package.